applications/SwimmingDEMApplication/custom_utilities/L2_error_projection_utility.h

```diff
@@ -111,6 +118,15 @@ double GetL2ScalarProjection(ModelPart& r_model_part)
         NContainer = rGeom.ShapeFunctionsValues(GeometryData::GI_GAUSS_2);
         const SizeType& NumGauss = NContainer.size1();
 
+        element_area = 0.0;
+
+        for (unsigned int k = 0; k < NumNodes; ++k)
+        {
+            element_area +=  ielem->GetGeometry()[k].FastGetSolutionStepValue(NODAL_AREA);
```

Comment:
The total volume is too big here... The sum of the nodal areas is the total area... But if you ad one node at a time! Here you are going element by element and then running through the nodes, so some nodes are counted many times